cocos/2d/CCScene.h

```diff
@@ -110,6 +110,7 @@ class CC_DLL Scene : public Node
      * @js NA
      */
     virtual void render(Renderer* renderer, const Mat4& eyeTransform, const Mat4* eyeProjection = nullptr);
+    virtual void render(Renderer* renderer, const Mat4* eyeTransforms, const Mat4* eyeProjections, unsigned int multiViewCount);
```

Comment:
this funtion will call only in MaliVR render for now.
It's hard to re-use, because the code is hard to break up